helmfile/values/networkpolicies/service/harbor.yaml.gotmpl

```diff
@@ -0,0 +1,324 @@
+        - rule: egress-rule-harbor-jobservice
+        - rule: egress-rule-harbor-registries
+
+    exporter:
+      podSelectorLabels:
+        component: exporter
+      ingress:
+        - rule: ingress-rule-prometheus
+          ports:
+            - tcp: 8001
+      egress:
+        - rule: egress-rule-dns
+        - rule: egress-rule-harbor-core
+        - rule: egress-rule-harbor-redis
+        - rule: egress-rule-harbor-database
+
+    {{- if eq .Values.harbor.database.type "internal" }}
+    database:
+      podSelectorLabels:
+        component: database
+      ingress:
+        - peers:
+          - podSelectorLabels:
+              component: core
+          - podSelectorLabels:
+              component: jobservice
+          - podSelectorLabels:
+              component: notary-server
+          - podSelectorLabels:
+              component: notary-signer
+          - podSelectorLabels:
+              component: registry
+          - podSelectorLabels:
+              component: exporter
+          - podSelectorLabels:
+              component: backup
+          {{- with .Values.networkPolicies.harbor.database.internalIngress }}
+          {{- if .peers }}
+          {{ toYaml .peers | nindent 10 }}
+          {{- end }}
+          {{- end }}
+          ports:
+          - tcp: 5432
+      egress:
+        - rule: egress-rule-dns
+      {{- end }}
+
+    {{- if eq .Values.harbor.redis.type "internal" }}
+    redis:
+      podSelectorLabels:
+        component: redis
+      ingress:
+        - peers:
+          - podSelectorLabels:
+              component: core
+          - podSelectorLabels:
+              component: jobservice
+          - podSelectorLabels:
+              component: trivy
+          - podSelectorLabels:
+              component: registry
+          - podSelectorLabels:
+              component: exporter
+          ports:
+          - tcp: 6379
+      egress:
+        - rule: egress-rule-dns
+      {{- end }}
+
+    registry:
+      podSelectorLabels:
+        component: registry
+      ingress:
+        - rule: ingress-rule-prometheus
+          ports:
+            - tcp: 8001
+        - peers:
+          - podSelectorLabels:
+              component: core
+          - podSelectorLabels:
+              component: jobservice
+          ports:
+            - tcp: 5000
+            - tcp: 8080
+      egress:
+        - rule: egress-rule-dns
+        - rule: egress-rule-object-storage
+        - rule: egress-rule-harbor-redis
+
+    portal:
+      podSelectorLabels:
+        component: portal
+      ingress:
+        - peers:
+          - podSelectorLabels:
+              component: core
+            ports:
+            - tcp: 8080
+        - rule: ingress-rule-ingress
+          ports:
+            - tcp: 8080
+      egress:
+        - rule: egress-rule-dns
+
+    notary-server:
+      podSelectorLabels:
+        component: notary-server
+      ingress:
+        - ports:
+            - tcp: 4443
```

Comment:
Should this be ingress-rule-ingress and any other harbor component that needs to talk to it?

Reply:
Ah yeah, missed this one. Will update.